web-common/src/components/editor/presets/yamlWithJsonAndSql.ts

```diff
@@ -0,0 +1,38 @@
+import { jsonLanguage } from "@codemirror/lang-json";
+import { SQLite } from "@codemirror/lang-sql";
+import { yamlLanguage } from "@codemirror/lang-yaml";
+import { LRLanguage } from "@codemirror/language";
+import { parseMixed } from "@lezer/common";
+
+const activateOnNodes = new Set(["BlockLiteralContent"]);
```

Comment:
Nit: a comment to explain where this "BlockLiteralContent" terminology is coming from would be helpful. Link to docs?

Reply:
There is no documentation around the new YAML parser. It was added couple of months ago. This mostly came through a hit and trial using the new parser.